L1Trigger/CSCTriggerPrimitives/src/CSCRPCMotherboard.h

```diff
@@ -0,0 +1,209 @@
+#ifndef CSCTriggerPrimitives_CSCRPCMotherboard_h
+#define CSCTriggerPrimitives_CSCRPCMotherboard_h
+
+/** \class CSCMotherboardME3141R0C
+ *
+ * Extended CSCMotherboard for ME3141 TMB upgrade
+ *
+ * \author Sven Dildick March 2014
+ *
+ * Based on CSCMotherboard code
+ *
+ */
+
+#include "L1Trigger/CSCTriggerPrimitives/src/CSCUpgradeMotherboard.h"
+#include "DataFormats/RPCDigi/interface/RPCDigiCollection.h"
+
+typedef match<RPCDigi>   RPCDigiId;
+typedef matches<RPCDigi> RPCDigiIds;
+typedef matchesBX<RPCDigi> RPCDigiIdsBX;
+
+class CSCGeometry;
+class CSCChamber;
+class RPCGeometry;
+
+class CSCRPCMotherboard : public CSCUpgradeMotherboard
+{
+ public:
+
+  /** Normal constructor. */
+  CSCRPCMotherboard(unsigned endcap, unsigned station, unsigned sector, 
+		    unsigned subsector, unsigned chamber,
+		    const edm::ParameterSet& conf);
+
+  /** Default destructor. */
+  ~CSCRPCMotherboard() override;
+
+  /** Run function for normal usage.  Runs cathode and anode LCT processors,
+      takes results and correlates into CorrelatedLCT. */
+  void run(const CSCWireDigiCollection* wiredc, 
+           const CSCComparatorDigiCollection* compdc, 
+           const RPCDigiCollection* rpcDigis);
+  
+  /// set CSC and RPC geometries for the matching needs
+  void setRPCGeometry(const RPCGeometry *g) { rpc_g = g; }
+  void setupGeometry();
+
+  /* store the CLCTs found earlier */
+  std::vector<CSCCLCTDigi> clctV;
+
+  /* readout the two best LCTs in this CSC */
+  std::vector<CSCCorrelatedLCTDigi> readoutLCTs() const;
+
+ private: 
+
+  /* access to the LUTs needed for matching */
+  const CSCRPCMotherboardLUT* getLUT() const {return tmbLUT_;}
+  const CSCRPCMotherboardLUT* tmbLUT_;
```

Comment:
Is this set or used anywhere?

Reply:
It is defined in the class CSCUpgradeMotherboardLUT and used in this class in the functions `matchingRPCDigis`